components/network/bind/patches/04-remove-zrelax-transtls.patch

```diff
@@ -0,0 +1,37 @@
+Revert this Solaris-specific patch.
```

Comment:
Do I understand correctly that this revers some gcc-related   tambourine dances which are not needed on illumos?

Reply:
Yes, I think so. Our `ld` does not know about `-zrelax=transtls` (and thus openssl link test fails) and our `ld` does not suffer from the problem described in https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90912.

I compiled and run the test case from there with our GCC 6..9 and all worked fine, see the output: [zrelax_transtls.test_case.txt](https://github.com/OpenIndiana/oi-userland/files/3480991/zrelax_transtls.test_case.txt).